syntax = "proto3";

import "bob_emploi/frontend/api/options.proto";

package bayes.bob;

message JobBoard {
  // Title of this job board.
  string title = 1;

  // Link to the job board.
  string link = 2 [(string_format) = URL_FORMAT];

  // A set of filter IDs: do not show the job board if one these
  // filters returns false for the project.
  repeated string filters = 3 [(string_format) = SCORING_MODEL_ID];

  // Whether this job board is well known.
  bool is_well_known = 4;
}

message JobBoards {
  // A sorted list of job boards. The ones that Bayes think are best suited are
  // first.
  repeated JobBoard job_boards = 1;
}
